contents/financial-aid.mdx

```diff
@@ -0,0 +1,54 @@
+# Financial Aid
+
+Our financial assistance program operates on a review basis, so please feel free to reach out if you are in need. After receiving all applications, we will make decisions based on factors such as the budget and the applicants' situations to determine whether assistance and to what extent it will be provided. Due to limited conference funds, the number of recipients will be limited. In this situation, we aim to prioritize those who are in the greatest need or can make the most significant contribution to the event. For example, students with lower family incomes will be given priority over engineers with stable employment.
+
+---
+
+## Eligibility
+
+Anyone in need is welcome to apply, and we strive to assist as many people as possible in participating in the conference.
+
+---
+
+## Auxiliary Items
+
+- sciwork 2024 Conference tickets
+- Transportation subsidies (the amount is determined based on the review results)
+
+---
+
+## Reimbursement
+
+sciwork 2024 Conference will send you a KKTIX discount code, which you can use to purchase tickets on KKTIX.
+
+---
+
+## Deadline
+
+From now until **December 13, 2024, at 23:59 (Taipei time)** is the application deadline.
```

Comment:
The application deadline is not same as application form, which is on `December 13, 2024`.

<img width="645" alt="image" src="https://github.com/user-attachments/assets/c7ef2678-6141-4103-8e2a-448d4f4f9bc4">

Reply:
Thanks @YenPeiChen07 I have corrected the deadline in [db95704](https://github.com/sciwork/swconf2024/pull/22/commits/db95704019a03450029bad1ad350eddb544ccb13) !